pygeoapi-deployment/pygeoapi.config.yml

```
@@ -536,6 +536,27 @@ resources:
         time_field: issue_time 
 
 
+  NOAA-RFC:
+    type: collection
+    title: NOAA RFC Forecasts
+    description: NOAA River Forecast Center Forecasts 
+    keywords: [NOAA]
+    links:
+      - type: application/html
+        rel: canonical
+        title: data source
+        href: https://data.usbr.gov/
```

Fix link. Naive of me to just take the data link if there is a better one to use. 
```suggestion
        href: https://www.cbrfc.noaa.gov/wsup/graph/west/map/esp_map.html
```